libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -698,8 +699,14 @@ internal class MapboxTripSession(
         }
     }
 
-    private fun invalidateLatestInstructions() {
-        bannerInstructionEvent.invalidateLatestBannerInstructions()
+    /**
+     * Invalidate latest banner instruction. To get the latest banner instruction wrapper call
+     * [BannerInstructionEvent.latestInstructionWrapper]
+     */
+    private fun invalidateLatestInstructions(
+        latestInstructionWrapper: BannerInstructionEvent.LatestInstructionWrapper?
+    ) {
+        bannerInstructionEvent.invalidateLatestBannerInstructions(latestInstructionWrapper)
```

Comment:
I guess voice instruction should be in sync, shouldn't it?

Reply:
what do you mean? They are removed only 🤔

Reply:
I mean that voice instruction follows the same mechanism as banner instruction. If you add additional logic for when not to invalidate banner instruction you probably need the same logic for voice instruction